CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## Unreleased
 
+IMPORTANT: Maven coordinates have been changed for some module. Be sure to update them, to get latest changes!
+
+| Old Maven coordinates                              | New Maven coordinates                                  |
+|----------------------------------------------------|--------------------------------------------------------|
+| com.pinterest.ktlint                               | com.pinterest.ktlint.ktlint-cli                        |
```

Comment:
I think the coordinates are actually with a double colon:

I.e.
```com.pinterest:ktlint` -> `com.pinterest.ktlint:ktlint-cli`

Reply:
Tnx. Fixed via #2264